### PR TITLE
Mention supported characters for mysql password in configuration page in docs service

### LINKED ISSUE #1274: Mention unsupported special characters for the mysql password
### What
https://lisk.com/documentation/lisk-service/references/configuration.html#internal-cache-persistence

### How
Update configuration reference: Mention unsupported special characters for the mysql password

### Additional information

So the final issue wound up being special characters in the ecosystem.core3.config.js file's mysql declaration

"URI malformed" is solved by removing special characters in mysql pass and declaration on ecosystem.core3.config.js 

### Affected branch
`v0.6` in `lisk-docs` repository

## Additional Information
https://dev.mysql.com/doc/refman/8.0/en/validate-password.html

### PR DESCRIPTION
**Pull request by chris529**
Closes #1274

### CODE REVIEW

File: modules/ROOT/pages/reference/configuration.adoc [hunk 1]
```diff
@@ -161,6 +162,9 @@ MYSQL_USER=lisk
 MYSQL_PASSWORD=password
 ----
 
+NOTE: The MYSQL_ROOT_PASSWORD and MYSQL_PASSWORD must not contain any special characters.
```
Comment (TalhaMaliktz) [inline modules/ROOT/pages/reference/configuration.adoc]: I would mention the special characters that are not supported by the `MYSQL_ROOT_PASSWORD` and `MYSQL_PASSWORD`. There might be some special characters that are supported.

Comment (chris529) [reply modules/ROOT/pages/reference/configuration.adoc]: Good point, now done 👍

File: modules/ROOT/pages/reference/configuration.adoc [hunk 2]
```diff
@@ -161,6 +162,14 @@ MYSQL_USER=lisk
 MYSQL_PASSWORD=password
 ----
 
+NOTE: The MYSQL_ROOT_PASSWORD and MYSQL_PASSWORD must not contain any of the following unsupported characters:
+Backslashes (\),
+Single quotes ('),
+Double quotes ("),
+Spaces, and
+Null bytes (\0).
+It is recommended to use alphanumeric characters and the commonly accepted special characters.
```
Comment (TalhaMaliktz) [inline modules/ROOT/pages/reference/configuration.adoc]:
```suggestion
NOTE: The `MYSQL_ROOT_PASSWORD` and `MYSQL_PASSWORD` must not contain any of the following unsupported characters:
Backslashes (\),
Single quotes ('),
Double quotes ("),
Spaces, and
Null bytes (\0).
It is recommended to use alphanumeric characters and the commonly accepted special characters.
```

**Review by TalhaMaliktz — APPROVED** [review]
Thanks, just a minor suggestion. The rest LGTM

File: modules/ROOT/pages/reference/configuration.adoc (hunk 2 above)
Comment (chris529) [reply modules/ROOT/pages/reference/configuration.adoc]: Ah yes, good point, now updated accordingly. 👍

File: modules/ROOT/pages/reference/configuration.adoc [hunk 3]
```diff
@@ -161,6 +162,14 @@ MYSQL_USER=lisk
 MYSQL_PASSWORD=password
 ----
 
+NOTE: The `MYSQL_ROOT_PASSWORD` and `MYSQL_PASSWORD` must not contain any of the following unsupported characters:
+Backslashes (\),
+Single quotes ('),
+Double quotes ("),
+Spaces, and
+Null bytes (\0).
```
Comment (nagdahimanshu) [inline modules/ROOT/pages/reference/configuration.adoc]: MySQL allows all ASCII characters for the password. So, I think we don't need to mention list of unsupported characters instead we can highlight all ASCII characters are allowed.

Comment (chris529) [reply modules/ROOT/pages/reference/configuration.adoc]: Updated, as discussed 👍

**Review by nagdahimanshu — APPROVED** [review]
LGTM 👍

**Pull request merged**